### ISSUE TITLE
Typo in Example 3.16

### ISSUE THREAD

**Issue by idg10** [stephencwelch/ai_book#10]
I couldn't find an official way to report errata for the book content, so I'm putting this here.

> 3.16 For the final training step 3, compute the model’s **our** $h$ and $\hat{y}$ values, cross-entropy loss, and final accuracy. Note that to compute accuracy you will have to pass all four training examples through the model. 

Note the spurious 'our' (which I've highlighted in **bold** above).
